api/v1alpha2/conversion_test.go

```diff
@@ -99,12 +114,18 @@ func TestConvertMachine(t *testing.T) {
 
 	t.Run("to hub", func(t *testing.T) {
 		t.Run("should convert the Spec.ClusterName from label", func(t *testing.T) {
+
 			src := &Machine{
 				ObjectMeta: metav1.ObjectMeta{
 					Labels: map[string]string{
 						MachineClusterLabelName: "test-cluster",
 					},
 				},
+				Spec: MachineSpec{
+					Bootstrap: Bootstrap{
+						ConfigRef: kubeAdmObj,
+					},
+				},
```

Comment:
I can't comment on the correct line, but please add this after the last expectation in this test case:

```go
g.Expect(dst.Spec.Bootstrap.ConfigRef.APIVersion).To(Equal(bootstrapv1a3.GroupVersion.String()))
```